Header header

bool calculates_speed_of_sound

bool depth

bool yaw

bool pitch

bool roll

bool salinity

bool temperature
